cds/base/bundles.py

```diff
@@ -29,12 +29,15 @@
 
 js = Bundle(
     "js/cds-settings.js",
-    "js/app.js",
     "js/main.js",
     output="cds.js",
     weight=91,
     filters="requirejs",
     bower={
         "es5-shim": "latest"
-    }
+    },
+    depends=[
+        'js/cds-settings.js',
+        'js/main.js'
+    ]
```

Comment:
Not sure about the _depends_ set here... http://webassets.readthedocs.org/en/latest/bundles.html

`require.config` (cds-settings.js) is only useful when you're not bundling.